cpu/esp32/esp-now/esp_now_netdev.c

```diff
@@ -236,24 +214,25 @@ static IRAM_ATTR void esp_now_recv_cb(const uint8_t *mac, const uint8_t *data, i
     }
 #endif
 
-    if (_esp_now_dev.rx_len) {
+    if (_esp_now_dev.rx_pkt.len) {
         /* there is already a packet in receive buffer, we drop the new one */
         return;
     }
 
     critical_enter();
 
 #if 0 /* don't printf anything in ISR */
-    printf ("%s\n", __func__);
-    printf ("%s: received %d byte from %02x:%02x:%02x:%02x:%02x:%02x\n",
-            __func__, len,
-            mac[0], mac[1], mac[2], mac[3], mac[4], mac[5]);
-    esp_hexdump (data, len, 'b', 16);
+    printf("%s\n", __func__);
+    printf("%s: received %d byte from %02x:%02x:%02x:%02x:%02x:%02x\n",
+           __func__, len,
+           mac[0], mac[1], mac[2], mac[3], mac[4], mac[5]);
+    od_hex_dump(data, len, OD_WIDTH_DEFAULT);
 #endif
 
-    _esp_now_dev.rx_len = len;
-    memcpy(_esp_now_dev.rx_buf, data, len);
-    memcpy(_esp_now_dev.rx_mac, mac, ESP_NOW_ADDR_LEN);
+    memcpy(&_esp_now_dev.rx_pkt, data, len);
+
```

Comment:
Is there any reason for this extra line?

Reply:
Not really, just got rid of it now.